src/HealthGPS/risk_factor_adjustable_model.h

```diff
@@ -55,6 +57,11 @@ class RiskFactorAdjustableModel : public RiskFactorModel {
     void adjust_risk_factors(RuntimeContext &context, const std::vector<core::Identifier> &factors,
                              OptionalRanges ranges, bool apply_trend) const;
 
+    /// @brief Gets the number of time steps to apply the trend
+    /// @param factor The risk factor to get the trend steps
+    /// @returns The number of time steps to apply the trend
+    int get_trend_steps(const core::Identifier &factor) const noexcept;
```

Comment:
This shouldn't be marked `noexcept`, because the `at()` method will throw a `std::out_of_range` if `factor` doesn't exist.